docs/api/model_selection/learning_curve.rst

```diff
@@ -3,6 +3,13 @@
 Learning Curve
 ==============
 
+=================   =====================
+ Visualizer           `LearningCurve <https://www.scikit-yb.org/en/latest/api/model_selection/learning_curve.html#yellowbrick.model_selection.learning_curve.LearningCurve>`_
+ Quick Method         `learning_curve() <https://www.scikit-yb.org/en/latest/api/model_selection/learning_curve.html#yellowbrick.model_selection.learning_curve.learning_curve>`_
```

Comment:
It might be a tad easier if we use the regular link syntax - this will ensure the links work for local builds, develop, latest, and our translations:

```
 Visualizer           :class:`~yellowbrick.model_selection.learning_curve.LearningCurve`
 Quick Method         :func:`~yellowbrick.model_selection.learning_curve.learning_curve`
```

Note that the `~` in front means that it doesn't display the whole path, just the name of the function or class.